src/cpp/function.h

```diff
@@ -18,9 +18,13 @@ class FunctionData : public GCData {
 
 class Function : public GCObject<FunctionData> {
 public:
-    sol::object loadFunction(lua_State* state);
+    sol::object loadFunction(lua_State* state) const;
+    sol::object convertToNative(lua_State* state, BaseHolder::DumpCache& cache) const;
 
 private:
+    using ConvertClbk = std::function<sol::object(const StoredObject&)>;
```

Comment:
This function can be named as `Converter`.

Reply:
ok